hr_worked_days/__manifest__.py

```diff
@@ -6,7 +6,17 @@
         Calculate working days excluding leaves in payslip""",
 
     'description': """
-        Long description of module's purpose
+        Calculation of working days
+        Calculates the number of working days for a selected month in the payslip calculation.
+        Some use cases where this module can be useful:
+            *an employee hasn't worked the whole month and his basic pay needs to be prorated
+        based on the actual worked days out of total worked days
+            *use working days information for calculation in any other salary rule
+        
+        A float field has been added to the payslip form view which is calculating the working 
+        days in respective working schedule of an employee by talking out the respective global
+        leaves.
+        
```

Comment:
@Yigitmm try to complete the rest part of the manifest too. Use the above link as a guide